<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-01T12:33:18 PST" name="Default suite" finished-at="2023-02-01T12:33:32 PST" duration-ms="14955">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.TestNGTestRunner@4b741d6d]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2023-02-01T12:33:18 PST" name="Default test" finished-at="2023-02-01T12:33:32 PST" duration-ms="14955">
      <class name="testRunner.TestNGTestRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:testRunner.TestNGTestRunner@4b741d6d]" started-at="2023-02-01T12:33:18 PST" name="setUpClass" finished-at="2023-02-01T12:33:19 PST" duration-ms="954" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@1d8b0500]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.TestNGTestRunner@4b741d6d]" started-at="2023-02-01T12:33:19 PST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-02-01T12:33:32 PST" duration-ms="13677" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Successfull login with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Verify Login"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at stepDefinitions.HomePageSteps.validate_if_login_is_successfull(HomePageSteps.java:31)
at âœ½.validate if login is successfull(file:///E:/Java_workspace/CucumberFramework/./Features/Login.feature:6)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:testRunner.TestNGTestRunner@4b741d6d]" started-at="2023-02-01T12:33:32 PST" name="tearDownClass" finished-at="2023-02-01T12:33:32 PST" duration-ms="227" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- testRunner.TestNGTestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
